packages/react-ui/src/layout/Wrapper/index.js

```diff
@@ -65,7 +65,14 @@ export const Wrapper = styled.div`
 Wrapper.propTypes = {
   children: PropTypes.node.isRequired,
   size: PropTypes.oneOf(["default", "large"]),
-  variant: PropTypes.oneOf(["dark", "default", "light", "main", "shadow"])
+  variant: PropTypes.oneOf([
+    "dark",
+    "darker",
```

Comment:
huhu darkerThanDark

Reply:
Damn, it was in the design, but we managed to remove it with Virginie and I forgot to delete this one :/

Reply:
Oh so we can remove it ? yeah!!!